src/gui/sdlmain.cpp

```diff
@@ -2824,10 +2826,11 @@ void Config_Add_SDL() {
 	Pstring->Set_help("What video system to use for output.");
 	Pstring->Set_values(outputs);
 
-	Pstring = sdl_sec->Add_string("texture_renderer", always, "auto");
-	Pstring->Set_help("Choose a renderer driver if output=texture or texturenb.\n"
-	                  "Use output=auto for an automatic choice.");
-	Pstring->Set_values(Get_SDL_TextureRenderers());
+	pstring = sdl_sec->Add_string("texture_renderer", always, "auto");
+	pstring->Set_help(
+	        "Choose a renderer driver if output=texture or texturenb.\n"
```

Comment:
Some very slight variations on this question:

- `"Choose a renderer driver when using a texture output mode"`
- `"Choose a renderer driver when using output=texture or texturenb"`
- `"Choose a renderer driver when using any of the texture output modes"`